grunttasks/scan-email-translations.js

```diff
@@ -0,0 +1,115 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.registerTask('scan-email-translations', 'Scan the generated emails for invalid style tags', [
+    'generate-valid-styles',
+    'po2json',
+    'verify-styles'
+    ]
+  );
+
+  grunt.registerMultiTask('verify-styles', 'Verify whether the translated styles are valid', function (){
+    var options = this.options({
+      validStyles: []
+    });
+    var styleArray = options.validStyles;
+
+    var fileArray = this.files[0].src;
+
+    var hasErrors = 0;
+    var styleRegex = /style="[\s\S]*?"/ig;
+
+    fileArray.forEach(function (src) {
+      var json = grunt.file.readJSON(src);
+      var styleList = [];
```

Comment:
This styleList is from where, is this the list of allowed styles or the list of styles from the translations? Can you use a more specific name?